region/maridia/inner-green.json

```diff
@@ -1464,6 +1592,25 @@
                     "Lateral Midair Morph for horizontal momentum, and perform the springball jump the moment before touching the sandfall.",
                     "Pause again to disable springball as soon as possible."
                   ]
+                },
+                {
+                  "name": "East Sand Hall Spring Ball Evir Freeze (Center to Right)",
```

Comment:
HiJumpless is a bigger feature than springball.  You'd still want to list both here or in the note.
"Suitless Evir Freeze" could use a name update too now.